ChainReactionBindings/Bindings/WishlistPageBindings.cs

```diff
@@ -86,5 +86,38 @@ public void WhenIClickConfirmButton()
 		{
 			page.ConfirmDelete();
 		}
+
+		[Then(@"I see empty wishlist with message ""(.*)""")]
+		public void ThenISeeEmptyWishlist(string message)
+		{
+			Assert.AreEqual(message, page.EmptyWishlistElement.Text);
+		}
+
+		[Then(@"I see ""(.*)"" items in wishlist")]
+		public void SeeItemsInWishlist(int count)
+		{
+			Assert.AreEqual(count, page.WishlistContainerElement.FindElements(By.ClassName("wishlist_product_container")).Count);
+		}
+
+		[When(@"I click Remove button on item with name ""(.*)""")]
+		public void ClickRemoveButtonOnItemWithName(string name)
+		{
+			var el = page.FindItem(name);
+			page.RemoveItem(el);

```

Comment:
I would suggest to use FindItem inside of RemoveItem, so it can be just 
page.RemoveItem(name);

Reply:
done